robot/markers.py

```diff
@@ -168,9 +168,12 @@ def spherical(self) -> SphericalCoord:
         """
         return SphericalCoord(*self._raw_data['spherical'])
 
+    def __repr__(self):
+        return "<robot.markers.{} >".format(str(self))
```

Comment:
I think replacing `robot.markers.` with nothing will lead to a nicer output. `<Marker 1: ... m away>` looks quite nice